docs/substrate/constants.md

```diff
@@ -1,6 +1,6 @@
 ## Constants
 
-The following sections contain the module constants, also known as parameter types. These can only be changed as part of a runtime upgrade. On the api, these are exposed via `api.consts.<module>.<method>`. 
+The following sections contain the module constants, also known as parameter types. These can only be changed as part of a runtime upgrade. On the api, these are exposed via `api.consts.<module>.<method>`.
```

Comment:
These docs are all auto-generated from the metadata. For the intro, revert this and make the relevant fixes here - https://github.com/polkadot-js/api/blob/master/packages/types/src/scripts/MetadataMd.ts